docs/installation.rst

```diff
@@ -42,6 +42,28 @@ For those who prefer to install PyBOP from a local clone of the repository or wi
 
 In editable mode, changes you make to the source code will immediately affect the PyBOP installation without the need for reinstallation.
 
+Optional Packages
+-----------------
+plotly - PyBOP uses plotly>=5.0, and can be installed with:
+
+.. code-block:: console
+
+    pip install plotly
+
+scikit-fem - This is a dependency for the multi-dimensional pybamm models, and can be installed using:
+
+.. code-block:: console
+
+    pip install scikit-fem[all]
+
+bpx - To use physics based battery models of Faraday Institution's MSM project Battery Parameter eXchange (BPX) is required:
+
+.. code-block:: console
+
+    pip install bpx
```

Comment:
and:
```suggestion
    pip install pybop[bpx]
```

Reply:
Thanks @BradyPlanden , shall update all these.